src/common/procedure/src/local.rs

```diff
@@ -291,12 +293,16 @@ impl ManagerContext {
 pub struct ManagerConfig {
     /// Object store
     pub object_store: ObjectStore,
+    pub max_retry_times: usize,
+    pub retry_delay: u64,
```

Comment:
suggestion: Just use Duration
```suggestion
    pub retry_delay: Duration,
```